resources/tools/index.md

```diff
@@ -61,6 +61,7 @@ meta_descr: meta_descr.tools
                     <div class="col">
                         <h2>{% t tools.market %}</h2>
                         <p><a href="https://cryptoradar.co/buy-monero">Cryptoradar - {% t tools.cryptoradar %}</a></p>
+                        <p><a href="https://kycnot.me/?t=exchange">KYCnot.me - {% t tools.kycnotme %}</a></p>
```

Comment:
Since this is tools page, maybe should just filter for xmr


```suggestion
                        <p><a href="https://kycnot.me/?t=&q=&xmr=on">KYCnot.me - {% t tools.kycnotme %}</a></p>
```


If perhaps we should also add to exchanges page with the exchange filter?